test/e2e/util/util.go

```diff
@@ -294,6 +294,22 @@ func EnsureNoCrashingPods(t *testing.T, ctx context.Context, client crclient.Cli
 	})
 }
 
+func NoticePreemptionOrFailedScheduling(t *testing.T, ctx context.Context, client crclient.Client, hostedCluster *hyperv1.HostedCluster) {
+	t.Run("NoticePreemptionOrFailedScheduling", func(t *testing.T) {
+		namespace := manifests.HostedControlPlaneNamespace(hostedCluster.Namespace, hostedCluster.Name).Name
+
+		var eventList corev1.EventList
+		if err := client.List(ctx, &eventList, crclient.InNamespace(namespace)); err != nil {
+			t.Fatalf("failed to list events in namespace %s: %v", namespace, err)
+		}
+		for _, event := range eventList.Items {
+			if event.Reason == "FailedScheduling" || event.Reason == "Preempted" {
+				t.Logf("error: non-fatal, observed FailedScheduling or Preempted event: %s", event.Message)
```

Comment:
`error: ` is to trigger prow syntax highlight
https://github.com/openshift/release/blob/d61b6d2ee203cdd872ddf75e9b8658836cb13c42/core-services/prow/02_config/_config.yaml#L54

Reply:
probably useful to put that in a code comment instead